tests/unit/package_managers/yarn/test_project.py

```diff
@@ -15,28 +15,29 @@
 )
 from cachi2.core.rooted_path import PathOutsideRoot, RootedPath
 
+# the supportedArchitectures property is not currently mapped in the YarnRc class, but is kept
```

Comment:
I know... it was generated by default in the projects I bootstraped locally. Maybe we can still avoid loading it, since we won't use it all in Cachi2.